### PR TITLE
Fix issues related to list `BBE`s

### PR DESCRIPTION
**Pull request by rdulmina**
1. Update incorrect `.bal` and `.out` references in the list related BBE `.md` files
2. Remove incorrect line breaks in metatags

### CODE REVIEW

File: examples/list-equality/list_equality.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 We can use `==` and `!=` on lists to check deep equality of two lists: two lists are deep equal if they have the same members in the same order. Deep equality only works for `anydata` lists. `===` and `!==` check for the exact equality which matches the references of the lists.
```
Comment (praneesha) [inline examples/list-equality/list_equality.md:3]:
```suggestion
You can use `==` and `!=` on lists to check the deep equality of two lists: two lists are deep equal if they have the same members in the same order. Deep equality only works for `anydata` lists. `===` and `!==` check for the exact equality, which matches the references of the lists.
```

File: examples/list-equality/list_equality.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 
-::: code updating_a_list.bal :::
+::: code list_equality.bal :::
 
-::: out updating_a_list.out :::
+::: out list_equality.out :::
 
 ## Related links
 - [Tuples - Ballerina by example](https://ballerina.io/learn/by-example/tuples)
```
Comment (praneesha) [inline examples/list-equality/list_equality.md:10]:
```suggestion
- [Tuples](https://ballerina.io/learn/by-example/tuples)
```

File: examples/list-subtyping/list_subtyping.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 An array with a length is a subtype of an array of same member type descriptor without a length. Two arrays with the same length or  without a length is a subtype of the other if the member type descriptor is a subtype of the other. Tuple type `T` is a subtype of  type `T1` if the set of values allowed in `T` is a sub set of values allowed in `T1`
```
Comment (praneesha) [inline examples/list-subtyping/list_subtyping.md:3]:
```suggestion
An array with a length is a subtype of an array of the same member type descriptor without a length. Two arrays with the same length or without a length are a subtype of the other if the member type descriptor is a subtype of the other. Tuple type `T` is a subtype of  type `T1` if the set of values allowed in `T` is a subset of the values allowed in `T1`
```

File: examples/list-equality/list_equality.md (hunk 2 above)
Comment (rdulmina) [reply examples/list-equality/list_equality.md:10]: IMO its better to keep the existing one to align with the `- Specification` and `- API documentation` suffixes in the existing BBEs

Comment (gimantha) [reply examples/list-equality/list_equality.md:10]: @praneesha @hasithaa shouldnt we have any indication of what type of related link this is? for example, BBE, Spec orGuide ?

Comment (rdulmina) [reply examples/list-equality/list_equality.md:10]: Please refer https://ballerina.io/learn/by-example/websocket-service-payload-constraint-validation/

**Review by hasithaa — APPROVED** [review]

**Pull request merged**